packages/patterns/README.md

```diff
@@ -1,103 +1,32 @@
 # `@endo/patterns`
 
-Builds on [`@endo/pass-style`](https://www.npmjs.com/package/@endo/pass-style) as described in [`kindOf` and `passStyleOf` levels of abstraction](./docs/marshal-vs-patterns-level.md) to define higher level data types as individual refinements of Passable CopyTagged records (PassStyle "tagged"):
-   - CopySet -- a collection of unique distinguishable Keys
-   - CopyBag -- a collection of entries associating a unique distinguishable Key with a positive integer count (see [Multiset](https://en.wikipedia.org/wiki/Multiset)).
-   - CopyMap -- a collection of entries associating a unique distinguishable Key with a Passable
-   - Matcher -- a predicate characterizing a subset of Passables, such as "strings" or "8-bit unsigned integer numbers" or "CopyArrays of Remotables"
+The main export from the package is an {@link M} namespace object, for making a variety of Matchers (hence "M"). See {@link PatternMatchers} for relevant methods.
```

Comment:
I'll guess that this `{@link M}` form is better for the generated documentation, which is great. But when viewed directly on github it does not render as a link or anything useful. Especially a README.md file should be pleasantly readable directly on github.

Can we get both somehow?

Reply:
We can write some links one way and some the other. I can sort of imagine demoting the `{@link X}` things so that they don't get in the way in non-typedoc contexts.